app/Helpers/database/player-achievement.php

```diff
@@ -40,74 +40,64 @@ function playerHasUnlock(?string $user, int $achievementId): array
     return $retVal;
 }
 
-function unlockAchievement(string $user, int $achIDToAward, bool $isHardcore): array
+function unlockAchievement(string $username, int $achIDToAward, bool $isHardcore): array
```

Comment:
`int $achievementId` for simpler parameter naming?

Reply:
I plan to follow this up with a migration to a command. I'll make the change there.

Reply:
Looks like there's already a partially implemented `UnlockPlayerAchievementAction` using the new schemas. I'll make the change here.